src/screens/introduction/index.js

```diff
@@ -2,47 +2,48 @@ export function Introduction() {
   return (
     <section
       id="introduction"
-      className="min-h-screen bg-savanna-md bg-cover bg-center py-24"
+      className="bg flex min-h-screen items-end bg-black bg-savanna bg-[length:100%] bg-[center_-100px] bg-no-repeat lg:items-center lg:bg-savanna-md lg:bg-cover lg:bg-center"
     >
-      <div className="mx-auto flex w-full max-w-7xl flex-col px-10">
-        <div className="grid grid-cols-2 gap-10">
-          <div className="p-12 pr-0">
+      <div className="mx-auto flex w-full max-w-7xl flex-col lg:gap-20 xl:gap-0">
+        <div className="grid lg:grid-cols-2">
+          <div className="mx-auto p-6 pt-12 pr-0 pb-0 lg:p-12 lg:pr-0">
```

Comment:
Instead of resetting pr and pb, we can set* pt and pl
```suggestion
          <div className="mx-auto pl-6 pt-12 lg:p-12 lg:pr-0">
```